### PR TITLE
chore: update telegraf plugins conf files to match new release

### PR DESCRIPTION
**Pull request by TCL735**
Changes from running `yarn run telegraf-plugins:update`

Telegraf just released version `1.20.4` with some minor updates to the configuration text of some plugins.

### CODE REVIEW

File: src/writeData/components/telegrafInputPluginsConfigurationText/nvidia_smi.conf [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Pulls statistics from nvidia GPUs attached to the host
 [[inputs.nvidia_smi]]
-  ## Optional: path to nvidia-smi binary, defaults to $PATH via exec.LookPath
+  ## Optional: path to nvidia-smi binary, defaults "/usr/bin/nvidia-smi"
```
Comment (hoorayimhelping) [inline src/writeData/components/telegrafInputPluginsConfigurationText/nvidia_smi.conf:3]:
```suggestion
  ## Optional: path to nvidia-smi binary, defaults to "/usr/bin/nvidia-smi"
```

not sure if this is what you were going for or not...

**Review by hoorayimhelping — APPROVED** [review]

**Review by sjwang90 — APPROVED** [review]
nice!

File: src/writeData/components/telegrafInputPluginsConfigurationText/nvidia_smi.conf (hunk 1 above)
Comment (TCL735) [reply src/writeData/components/telegrafInputPluginsConfigurationText/nvidia_smi.conf:3]: It makes sense to add the "to" there but this change was pulled from the Telegraf repository. The change should be made there.

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 60 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3280)

**Pull request merged**